packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -87,3 +82,58 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
     />
   );
 };
+
+/**
+ * Hook to override the avatar persona data of the users
+ * @private
+ */
+export const useDataProvider = (
+  userIds: (string | undefined)[],
+  dataProvider?: AvatarPersonaDataCallback
```

Comment:
nit: rename `dataProvider`. Even just `callback` reads better than `dataProvider` to me.